### DISCUSSION TITLE
How to translate Admin pages to Arabic?

### DISCUSSION THREAD

**Discussion by HannahHatem-SK** [OrchardCMS/OrchardCore#10518] [Q&A] [answered]
I typed in the URL ➡️ https://localhost:5001/admin/Features?culture=ar
![image](https://user-images.githubusercontent.com/87015553/137896415-9d2c72f1-0caf-48ec-a789-c1a3e6d078c2.png)

- This translates **some** of the content on this page, but I need to translate **all** Admin Pages.
@hishamco

**Comment by hishamco** [comment, marked as answer]
@HannahHatem-SK you could add Arabic language by enabling `Localization` module, then go to settings -> cultures, after that you can add `ar` or `ar-XX` culture

Regarding your question, there are some missing translations, it would be nice if you want to contribute via https://crowdin.com/

**Reply by HannahHatem-SK** [reply]
> Regarding your question, there are some missing translations, it would be nice if you want to contribute via https://crowdin.com/

@hishamco How can I contribute to the translation?

**Reply by hishamco** [reply]
Please sign up into Crowdin https://crowdin.com/project/orchard-core, then choose Arabic, after that you can add missing translations, I will try to add some too

Thanks
